formatTest/typeCheckedTests/expected_output/oo.re

```diff
@@ -157,21 +157,24 @@ let acceptsClosedAnonObjAsArg
     (o: < x : int, y : int >) =>
   o#x + o#y;
 
-let res = acceptsOpenAnonObjAsArg {
-  method x = 0;
-  method y = 10
-};
+let res =
+  acceptsOpenAnonObjAsArg {
+    method x = 0;
+    method y = 10
+  };
 
-let res = acceptsOpenAnonObjAsArg {
```

Comment:
I think the old version is more conventional (FB's conventions at least).

```rust
let myComponent = React.CreateComponent {

};
```

Because it's under `50` characters, it shouldn't have to break to the next line.